FSharp-6.0/FS-1087-resumable-code.md

```diff
@@ -1024,7 +1024,7 @@ Next we define the `Run` method for the builder.
                     cr.Machine <- sm
                     cr :> Coroutine))
         else 
-            failwith "dynamic implementation nyi"
+            failwith "dynamic implementation NYI"
```

Comment:
I'm assuming this is `Not-Yet-Implemented`

Reply:
Using lowercase `nyi` is a common acronym, it should be kept.